libres/tests/enkf/test_enkf_fs.cpp

```diff
@@ -93,7 +92,15 @@ TEST_CASE("block_fs", "[enkf_fs]") {
             AND_WHEN("block_fs is closed and opened") {
                 block_fs_close(bfs, false /* unlink_empty */);
                 bfs = block_fs_mount("bfs", block_size, fragmentation_limit,
-                                     fsync_interval, read_only);
+                                     fsync_interval, true /* read-only */);
```

Comment:
true here, false above?

Reply:
Changed the reopened bfs to be read-only so that the new test afterwards is correct. The test checks that you get an exception (previously `util_abort`) if you try to write to a readonly bfs.

Reply:
ok